src/eko/evolution_operator/__init__.py

```diff
@@ -688,27 +688,27 @@ def compute_aem_list_as(self):
         q2ref = self.managers["couplings"].q2_ref
         delta_from = abs(self.q2_from - q2ref)
         delta_to = abs(self.q2_to - q2ref)
-        # I compute the values in aem_list_as starting from the as
-        # that is closer to as_ref.
+        # I compute the values in aem_list starting from the mu2
+        # that is closer to mu_ref.
         if delta_from > delta_to:
-            as_start = as1
-            aem_start = aem1
+            a_start = np.array([as1, aem1])
+            mu2_start = self.q2_to
         else:
-            as_start = as0
-            aem_start = aem0
+            a_start = np.array([as0, aem0])
+            mu2_start = self.q2_from
         sc = self.managers["couplings"]
         ev_op_iterations = self.config["ev_op_iterations"]
-        as_steps = utils.geomspace(as0, as1, 1 + ev_op_iterations)
-        as_l = as_steps[0]
+        mu2_steps = np.linspace(self.q2_from, self.q2_to, 1 + ev_op_iterations)
```

Comment:
Why did you change to `linspace`? the natural variable in evolution is $\ln(Q^2)$ (look at LO alpha evolution)

Reply:
I agree, but I have to use it in `sc.compute` (few lines after) that wants a $\mu_2$

Reply:
Mh? I don't understand ... `geomspace` is just saying how to distribute the values; so whether you have a list `[1,2,4]` or `[1,2.5,4]` ... the actual element has always the same unit ($\mu^2$ to be specific)

Reply:
Ah ok I thought that you wanted to divide in steps $\log(\mu^2)$ instead of $\mu^2$.
I did it since the i-th $a_{em}$ value must be in the interval $a_s^{(i+1)} - a_s^{(i)}$, and since the $a_s$ values are logspaced then $\mu^2$ should be linspaced.